tests/data_platform/ingestion/test_sync_twitter_checkpoint.py

```
@@ -398,3 +398,95 @@ def fake_fetch(
+        [
+            [],
+            ["twitter.user"],
+            "twitter.tweet",
+            None,
+        ],
+    )
+    def test_rejects_empty_or_wrong_record_types(
+        self,
+        data_root,
+        monkeypatch: pytest.MonkeyPatch,
+        record_types: Any,
```

Ruff (0.16.3)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `record_types`

(ANN401)